tests/integration/test_storage_rabbitmq/configs/rabbitmq.xml

```diff
@@ -1,6 +1,6 @@
 <clickhouse>
     <rabbitmq>
-        <username>root</username>
-        <password>clickhouse</password>
+        <username>rabbitmq</username>
+        <password>ClickHouse_RabbitMQ_P@ssw0rd</password>
```

Comment:
it looks like creds from config are no longer tested since we pass `rabbitmq_password` explicitly?